Signal/src/call/CallService.swift

```diff
@@ -713,7 +715,7 @@ fileprivate let timeoutSeconds = 60
      *
      * Can be used for Incoming and Outgoing calls.
      */
-    func handleToggledMute(isMuted: Bool) {
+    func setIsMuted(isMuted: Bool) {
```

Comment:
There's a lot of "toggle mute" methods that are actually setters.  I've renamed them appropriately.